CMake/SlicerConfigurevtkSlicerVersionConfigure.cmake

```diff
@@ -0,0 +1,60 @@
+################################################################################
+#
+#  Program: 3D Slicer
+#
+#  Copyright (c) Kitware Inc.
+#
+#  See COPYRIGHT.txt
+#  or http://www.slicer.org/copyright/copyright.txt for details.
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+#  This file was originally developed by Jean-Christophe Fillion-Robin, Kitware Inc.
+#  and was partially funded by NIH grant 3P41RR013218-12S1
+#
+################################################################################
+
+
+#-----------------------------------------------------------------------------
+# Slicer version number.
+#-----------------------------------------------------------------------------
+# Releases define a tweak level
+#set(Slicer_VERSION_TWEAK 1)
+#set(Slicer_VERSION_RC 0)
```

Comment:
I like to add a `Sanity check` section like it is done here: https://github.com/Slicer/Slicer/blob/bd4cccebd71d124927181f57d5ba1732c317d3cd/CMake/SlicerMacroConfigureModuleCxxTestDriver.cmake#L43-L51